### PR TITLE
Fix Matpower importer performance issue

### PR DESCRIPTION
**Pull request by geofjamg**
Signed-off-by: Geoffroy Jamgotchian <geoffroy.jamgotchian@rte-france.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Bug fix

**What is the current behavior?** *(You can also link to an open issue here)*
On large network, import is slow because of missing index for generators by bus number.

**What is the new behavior (if this is a feature change)?**
Matpower import is 50% faster.

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: matpower/matpower-converter/src/main/java/com/powsybl/matpower/converter/MatpowerImporter.java [hunk 1]
```diff
@@ -124,7 +121,8 @@ private static void createBuses(MatpowerModel model, ContainersMapping container
     }
 
     private static void createGenerators(MatpowerModel model, MBus mBus, VoltageLevel voltageLevel) {
-        model.getGenerators().stream().filter(gen -> gen.getNumber() == mBus.getNumber()).forEach(mGen -> {
+        MGen mGen = model.getGeneratorByBusNum(mBus.getNumber());
```
Comment (miovd) [inline matpower/matpower-converter/src/main/java/com/powsybl/matpower/converter/MatpowerImporter.java:124]: Just a question: is it possible to have several generators to the same bus?

Comment (geofjamg) [reply matpower/matpower-converter/src/main/java/com/powsybl/matpower/converter/MatpowerImporter.java:124]: It's seems yes, even if not seen in our cases. Thanks I'm going to fix it.

Comment (geofjamg) [reply matpower/matpower-converter/src/main/java/com/powsybl/matpower/converter/MatpowerImporter.java]: Fixed

**Review by miovd — APPROVED** [review]
Small remark: maybe rename `getGeneratorByBusNum` as `getGeneratorsByBusNum`. Otherwise, okay for me!

**Comment by geofjamg** [comment]
> Small remark: maybe rename `getGeneratorByBusNum` as `getGeneratorsByBusNum`. Otherwise, okay for me!

Done

**Pull request merged**